app/src/main/java/plantfueled/puppysitter/Pet.java

```diff
@@ -11,6 +13,10 @@
     private float hungerLevel = 100;
     private float lonelyLevel = 100;
     private Date lastPetUpdate;
+    private PetNotification petNotification;
+
+    final float HUNGER_RATE = 10; // points per minute lost
+    final float LONELY_RATE = 15; // points per minute lost
```

Comment:
These can be public static as well

Reply:
Why make these public static? I don't think the rates will be used by anything else.

Reply:
Alright then, but atleast make them private in that case

Reply:
and static

Reply:
static because those floats are constant, if you put it static the constants will be stored in the class instead of each individual object